samples/logging/application-insights/sample.md

```diff
@@ -1,7 +1,16 @@
 ---
 title: Report metrics to Azure Application Insights
+summary: Shows how to report NServiceBus metrics data to Azure Application Insights
 component: Metrics
-reviewed: 2017-07-29
+reviewed: 2017-10-06
+tags:
+- Azure
+- Metrics
```

Comment:
Content verification says "Each tag must have more than one page" ... guessing Metrics is the one that has only one. Add it to other related pages or remove the tag.